pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -1917,7 +2083,7 @@ mod benchmarks {
 
 		// initialize our single collator
 		let sole_collator =
-			create_funded_collator::<T>("collator", 0, initial_stake_amount, true, 1u32)?;
+			create_funded_collator::<T>("collator", 0, initial_stake_amount, true, 1u32, false)?;
```

Comment:
Can you explain why we can assume here that the collator account is already migrated?

Reply:
This was an oversight. Amended https://github.com/moonbeam-foundation/moonbeam/pull/3306/commits/3cd5365a8a80a3ec9b0e3c2601fe4af49c3b0775